test/quality/.python-version

```diff
@@ -0,0 +1 @@
+3.10.7
```

Comment:
I'm not convinced this is a good thing to check in -- what if the user has `3.10.6` or some other functional `3.10` version, just not `3.10.7`?